Resources/Private/Templates/TechnicalHeadline.html

```
@@ -7,18 +7,18 @@
+        <span class="me-typo3-styleguide-technical-headline__title">{data.header}</span>
+        <a class="me-typo3-styleguide-technical-headline__to-top" href="#me-typo3-styleguide-technical-headline-toc" aria-label="{f:translate(key: 'technical_headline.headline.top.title', extensionName: 'metypo3styleguide')}">
             &uarr;
         </a>
     </{data.tx_metypo3styleguide_technicalheadlinetag}>
     <f:if condition="{data.subheader}">
-        <h3 class="technical-headline__label">{data.subheader}</h3>
+        <h3 class="me-typo3-styleguide-technical-headline__label">{data.subheader}</h3>
     </f:if>
     <f:if condition="{data.bodytext}">
-        <p class="technical-headline__text">{data.bodytext -> f:format.raw()}</p>
+        <p class="me-typo3-styleguide-technical-headline__text">{data.bodytext -> f:format.raw()}</p>
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)